ngraph/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -146,5 +146,25 @@ void PreProcessSteps::PreProcessStepsImpl::add_resize_impl(ResizeAlgorithm alg,
         true));
 }
 
+void PreProcessSteps::PreProcessStepsImpl::add_convert_layout_impl(const Layout& layout) {
+    m_actions.emplace_back(std::make_tuple(
+        [layout](const std::vector<std::shared_ptr<Node>>& nodes, PreprocessingContext& context) {
+            OPENVINO_ASSERT(!nodes.empty(), "Internal error: Can't convert layout for empty input.");
+            OPENVINO_ASSERT(nodes.size() == 1,
+                            "Can't convert layout for multi-plane input. Suggesting to convert current image to "
+                            "RGB/BGR color format using 'convert_color'");
+            Layout dst_layout = layout == Layout() ? context.network_layout() : layout;
```

Comment:
can we have `layout.empty()` for `layout == Layout()` ?

Reply:
Done